core/src/main/kotlin/fund/cyber/markets/dto/TokensPairDto.kt

```diff
@@ -0,0 +1,29 @@
+package fund.cyber.markets.dto
+
+/**
+ * @author mgergalov
+ */
+class TokensPairDto(
```

Comment:
maybe to not have name conflicts, rename this class to TokensPair
and current TokensPair to something more meaningful(also it can contains not just base, quote, but newly tokens pair)